psiminer-core/src/main/kotlin/psi/language/KotlinHandler.kt

```diff
@@ -16,10 +17,25 @@ class KotlinHandler : LanguageHandler() {
 
     override val transformationType = KotlinTreeTransformation::class.java
     override val classPsiType = KtClass::class.java
-    override val methodPsiType = KtNamedFunction::class.java
+    override val methodPsiType = KtFunction::class.java
 
-    override fun collectFunctionCallsIdentifiers(root: PsiElement): List<PsiElement> =
-        PsiTreeUtil
-            .collectElements(root) { it.elementType.toString() == "IDENTIFIER" }
-            .filter { it.parent is KtReferenceExpression }
+    private val KtExpression.resolveToElement: PsiElement?
```

Comment:
Why do you make it a field with a getter rather than a function?

Reply:
I like how it looks, from execution side, both expressions translate into the same bytecode